### ISSUE TITLE
KeyValue form component collapsing before can actually enter anything into the new rows when clicking add row

### ISSUE THREAD

**Issue by Cigoler** [laravel-filament/filament#7785]
### Package

filament/filament

### Package Version

v3.0.22

### Laravel Version

v10.10

### Livewire Version

v3

### PHP Version

PHP 8.1.0

### Problem description

When clicking the add row button the table flashes to indicate the creation of the row, but quickly collapses back to no rows.

### Expected behavior

A row to be created, and able to enter information into.

### Steps to reproduce

This repository should demonstrate it on the user create page when you login to the admin panel. I notice (on windows) if I use sail down that the page, if it's still loaded will then start working.

### Reproduction repository

https://github.com/Cigoler/fv3-keyvalue-issue

Link to discord help topic: https://discord.com/channels/883083792112300104/1140992574476267610

**Comment by steveallinson** [comment]
Can confirm this is an issue for us also after recent update. Video of the issue [here](https://www.awesomescreenshot.com/video/19956655?key=386c9687d4668f3ea9eccca69570d190). Using the KeyValue with a wizard.

**Comment by killjin** [comment]
Confirmed that there is no problem under livewire 3.0.0-beta.7, it seems that the problem of livewire changes

**Comment by killjin** [comment]
https://github.com/filamentphp/filament/pull/7754/commits/582568f8e9b164b8a80041d305181db5d19670cb

This request can solve the problem, please pass, I thought that the previous PR has been merged and put together.

**Comment by killjin** [comment]
change  filament/packages/forms/resources/js/components/key-value.js
```js
this.state = state 
```
to
```js
JSON.stringify(state) != "{}" && (this.state = state) 
```
can be solved

Livewire Below 3.0 beta7, the state change will not trigger the request immediately.

**Comment by Cigoler** [comment]
Well it was working with filament v3.0.19 (had another instance on another PC with the older version), but soon as I upgraded to filament v3.0.22 it stopped working, but I'll try your fix - what version of livewire does v3.0.22 use? (It's livewire/livewire                            v3.0.0-beta.9 for those interested)

Edit: Oh I see, I misread what you were saying Killjin.

**Comment by killjin** [comment]
This may be a livewire bug, entangle is delayed by default, and now it is live.

**Comment by killjin** [comment]
https://github.com/filamentphp/filament/pull/7807  Fix

**Comment by wit3** [comment]
the bug is still in version v3.0.40 @danharrin

**Comment by danharrin** [comment]
Please check your Livewire version

**Comment by wit3** [comment]
> Please check your Livewire version

"livewire/livewire": "^3.0.2"

it's the latest 😕

**Comment by danharrin** [comment]
Fixed by #8386
